### ISSUE TITLE
Reproducing the training results

### ISSUE THREAD

**Issue by wkbian** [16lemoing/dot#12]
Hi, @16lemoing,

Congratulations on your paper acceptance! :tada:

I encountered some problems while reproducing your training results. I followed the instructions in [training](https://github.com/16lemoing/dot?tab=readme-ov-file#training) section. Seems the motion loss was not convergent while I set `world_size = 4` which aligns with the setting in the paper.
"DOT is trained on frames at resolution 512×512 for 500k steps with the ADAM optimizer [32] and a learning rate of 10−4 using 4 NVIDIA V100 GPUs."
<img width="380" alt="image" src="https://github.com/16lemoing/dot/assets/17591059/48553142-780c-403f-90be-0c6b73f6640c">
Could you please provide some suggestions? thx~

**Comment by 16lemoing** [comment]
Hi @wkbian, it is normal that the training loss is a bit noisy. Can you run the evaluation on CVO to properly evaluate the performance of the final model? For example:

```
python test_cvo.py --split final --refiner_path checkpoints/YOUR_RUN/last.pth
```

I have found a bug in the code with the distributed training mode: all the GPUs were sampling the same elements of the dataset simultaneously. The issue is solved in https://github.com/16lemoing/dot/commit/cdee971fb0615fe3bf7b6fd19d856ea572327ec1 

Also setting the flag `--lambda_motion_loss 1000` improves a bit motion prediction quality but degrades a bit visibility prediction. This is what we use in our final method.
